cmd/exo/cmd/sos_acl.go

```diff
@@ -105,6 +106,14 @@ var sosAddACLCmd = &cobra.Command{
 	},
 }
 
+//merge src header in dst header
+func mergeHeader(src, dst http.Header) http.Header {
+	for k, v := range src {
+		dst[k] = v
+	}
+	return dst
+}
```

Comment:
This this highly confusing.
- `src`, `dst` vs <https://golang.org/pkg/builtin/#copy>
- you modify `dst` **and** return it

Reply:
I want to merge two header. parameters order is wrong ?

Reply:
- `copy` uses `dst, src`
- coming from the Python world, you _either_ modify and returns `None`, or retuns the modified value.